shared/chat/conversation/messages/wrapper/wrapper-author/index.js

```diff
@@ -158,7 +158,7 @@ const RightSide = props => {
               isBroken={props.isBroken}
               onClick={props.onAuthorClick}
             />
-            <Text type="BodyTiny">{formatTimeForConversationList(props.timestamp)}</Text>
+            <Text type="BodyTiny">{moment(props.timestamp).format('h:mm A')}</Text>
```

Comment:
if we need another format we should just add it to util/timestamp (so it can be mocked also). we really don't want the existing one?

Reply:
Will do. I could exit early from the existing one or add a small helper. The existing one's logic changes the timestamp if it's older than a day.